### PR TITLE
refactor homePage

### PR DESCRIPTION
**Pull request by jeffery021121**
refactor homePage

### CODE REVIEW

**Review by yaooqinn — APPROVED** [review]

**Comment by yaooqinn** [comment]
👍

File: data/homePage.json [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+    },
+    {
+      "img": "homePageImg/community2.png",
+      "title": "Source code",
+      "href": "https://github.com/apache/incubator-kyuubi"
+    },
+    {
+      "img": "homePageImg/community3.png",
+      "title": "How to contribute",
+      "href": "https://kyuubi.apache.org/contributing.html"
```
Comment (cxzl25) [inline data/homePage.json:80]: This link accesses 404, should it point to https://kyuubi.apache.org/issue_tracking.html ?

Comment (ulysses-you) [reply data/homePage.json:80]: @cxzl25 good catch !

Comment (ulysses-you) [reply data/homePage.json:80]: do you have time to fix it ? thank you !

Comment (yaooqinn) [reply data/homePage.json:80]: maybe we shall use https://github.com/apache/incubator-kyuubi/blob/master/CONTRIBUTING.md

Comment (cxzl25) [reply data/homePage.json:80]: open a PR: https://github.com/apache/incubator-kyuubi-website/pull/52

**Pull request merged**